packages/daimo-contract/src/tokenMetadata.ts

```diff
@@ -1,5 +1,4 @@
-// Generated by script/fetchTokenMetadata.ts
-
+// TestUSDC on Base Goerli
```

Comment:
I would keep this comment as before. this file is autogenerated and should not be edited manually (after #255 it also generates paymaster variables)

Reply:
going to rebase and revert this fetch script deletion since paymaster stuff also uses that script now (and I dont see any other clean way to get the address).